--- /usr/lib/python3/dist-packages/glance_store/_drivers/cinder.py      2023-01-22 05:45:59.578500616 +0000
+++ /tmp/cinder.py      2023-01-22 05:46:22.154444687 +0000
@@ -991,7 +991,7 @@
                     # Sometimes the extended LUN on storage side takes time
                     # to reflect in the device so we wait until the device
                     # size is equal to the extended volume size.
-                    Store._wait_resize_device(volume, f)
+                    # Store._wait_resize_device(volume, f)
                     f.seek(bytes_written)
                     if buf:
                         f.write(buf)
